Source/ProgressionSystemRuntime/Public/Data/PSDataAsset.h

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "PSDataAsset.generated.h"
+
+/**
+ * Contains all progression assets used in the module 
+ */
+UCLASS()
+class PROGRESSIONSYSTEMRUNTIME_API UPSDataAsset : public UDataAsset
```

Comment:
Please include `#include "Engine/DataAsset.h"` as its parent class. Otherwise, it will fail the cook.

Reply:
done